src/test/java/com/wikia/webdriver/testcases/mediatests/videohomepage/VideoHomePageTests.java

```diff
@@ -1,26 +0,0 @@
-package com.wikia.webdriver.testcases.mediatests.videohomepage;
-
-import com.wikia.webdriver.common.contentpatterns.URLsContent;
-import com.wikia.webdriver.common.core.annotations.Execute;
-import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.pageobjectsfactory.pageobject.videohomepage.VideoHomePageObject;
-
-import org.testng.annotations.Test;
-
-public class VideoHomePageTests extends NewTestTemplate {
-
-  @Test(groups = {"VideoHomePage_001", "Media", "VideoHomePageTests", "VideosPage"})
```

Comment:
If this (or any other deleted) group was ever executed on Jenkins, please remember to also remove corresponding config from jenkins-jobs repo

Reply:
+1 ...and remember that executing `delete` command in jenkins-jobs won't delete all desired jobs (only one by one). To remove these jobs it's best to remove them via Jenkins console or manually (sic).